com.unity.cinemachine/Samples~/3D Samples/SplitScreenCar/SplitScreenCar.unity

```diff
@@ -0,0 +1,892 @@
+%YAML 1.1
```

Comment:
You should change one of the cars to blue, that way it would be easier to understand.  See the Brain Update Modes scene, it has a red and blue car.  Do it the same way.

Reply:
Fixed in latest commit